src/interface/storage.thrift

```diff
@@ -142,9 +158,9 @@ struct EdgePropRequest {
     1: common.GraphSpaceID space_id,
     // partId => edges
     2: map<common.PartitionID, list<EdgeKey>>(cpp.template = "std::unordered_map") parts,
-    3: common.EdgeType edge_type,
-    4: binary filter,
-    5: list<PropDef> return_columns,
+    3: binary filter,
+    4: list<PropDef> return_columns,
+    5: bool over_all_edges,
```

Comment:
Why we have over_all_edges in EdgePropRequest,  the edgeKeys have been specified.

Reply:
The main reason was that the vertex prop request also calls the QueryBaseProcessor<REQ, RESP>::checkAndBuildContexts method, which uses "req.over_all_edges".

Reply:
To be compatible,  we could do it temporarily,  but it is not a good reason. We should not design our interface as implementation.  Add some comments,  and we go on this pr.